BlueDBOnDisk/src/main/java/io/bluedb/disk/BlueDbOnDisk.java

```diff
@@ -63,12 +63,15 @@
 
 	@Override
 	public void backup(Path zipPath) throws BlueDbException {
```

Comment:
Do we have any protection against them doing multiple backups at the same time? Maybe that's not worth doing right now. Would it cause problems? At the very least, the first backup would change all of the retention periods back to default while the second one was still running.